files/etc/uci-defaults/96_aredn_migrate_latlon

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+if [ "$(/sbin/uci -c /etc/config.mesh -q get aredn.@location[0])" = "" ]; then
+    /sbin/uci -c /etc/config.mesh add aredn location
+    /sbin/uci -c /etc/config.mesh commit aredn
+fi
+
+# read /etc/latlon
+if [ -f /etc/latlon ]
+then
+    LAT=$(head -1 /etc/latlon)
+    LON=$(tail -1 /etc/latlon)
+fi
+
+if [ -f /etc/gridsquare ]
+then
+    GRIDSQUARE=$(head -1 /etc/gridsquare)
+fi
+
+/sbin/uci -c /etc/config.mesh -q set aredn.@location[0].lat="$LAT"
+/sbin/uci -c /etc/config.mesh -q set aredn.@location[0].lon="$LON"
+/sbin/uci -c /etc/config.mesh -q set aredn.@location[0].gridsquare="$GRIDSQUARE"
+/sbin/uci -c /etc/config.mesh -q commit aredn
+
+cp /etc/config.mesh/aredn /etc/config/aredn
+
+# remove the /etc/latlon and gridsquare files
+rm /etc/latlon
+rm /etc/gridsquare
```

Comment:
Add '-f' flags here because these files might not exists. Without -f this script with terminate with an error (causing it to be rerun on the next boot .. and the next .. etc.)